assets/js/util/scroll.js

```diff
@@ -58,11 +50,36 @@ export function getContextScrollTop( context, breakpoint ) {
 	const anchorAdjustment =
 		context === '#googlesitekit-pagespeed-header' ? 80 : 0;
 
-	return (
-		contextTop +
-		global.scrollY -
-		headerHeight -
-		navigationHeight -
-		anchorAdjustment
+	return contextTop + global.scrollY - headerHeight - anchorAdjustment;
+}
+
+/**
+ * Gets the height of the sticky header.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} breakpoint The current breakpoint.
+ * @return {number} The height of the sticky header.
+ */
+export function getHeaderHeight( breakpoint ) {
```

Comment:
Another trivial detail - the IB mentions creating this function in a new file, `assets/js/util/header.js`. Again, it's fine by me if you prefer to leave it in `scroll.js`.